build.fsx

```diff
@@ -97,12 +97,33 @@ let freya =
                           Local "Freya.Types.Http"
                           Local "Freya.Types.Language"
                           Local "Freya.Types.Uri" ] }
+                  { Name = "Freya.Machine.Extensions.Http"
+                    Dependencies =
+                        [ Package "FSharp.Core"
+                          Package "Aether"
+                          Local "Freya.Core"
+                          Local "Freya.Machine"
+                          Local "Freya.Types.Http"
+                          Local "Freya.Types.Language"
+                          Local "Freya.Types.Uri" ] }
+                  { Name = "Freya.Machine.Extensions.Http"
```

Comment:
This is the same name as the extensions above, isn't it? Is this meant to be `Freya.Machine.Extensions.Cors`?